tgui/packages/tgui/interfaces/TTV.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Romayne (https://github.com/MeggalBozale)
+ * @license ISC
+ */
+
+import { useBackend } from '../backend';
+import { Button, LabeledList, RoundGauge, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { toTitleCase } from '../../common/string';
+import { formatPressure } from '../format';
+interface AirVendorParams {
+  opened: boolean;
+  tank_one: TankData;
+  tank_two: TankData;
+  device: string;
+}
+
+interface TankData {
+  name: string;
+  num: number;
+  pressure: number;
+  maxPressure: number;
+}
+
+const TankInfo = (_props, context) => {
+  const { act, data } = useBackend<AirVendorParams>(context);
+  const { tank, height } = _props;
+  let button_eject = <Button width={5} textAlign={"center"} disabled={tank.name===null} icon="eject" onClick={() => act(tank.num === 1 ? "remove_tank_one" : "remove_tank_two")}>Eject</Button>;
+  let button_add = <Button width={5} textAlign={"center"} icon="add" onClick={() => act("add_item", { "tank": tank.num })}>Add</Button>;
+  let maxPressure = (tank.maxPressure !== null) ? tank.maxPressure : 999;
+  return (
+    <Section
+      title={tank.num === 1 ? "Tank One" : "Tank Two"}
+      buttons={tank.name !== null ? button_eject : button_add}
+      height={height}
+    >
+      <LabeledList>
+        <LabeledList.Item label={"Holding"}>
+          {tank.name !== null ? toTitleCase(tank.name) : "None"}
+        </LabeledList.Item>
+        <LabeledList.Item
+          label="Pressure">
+          <RoundGauge
+            size={1.75}
+            value={tank.pressure !== null ? tank.pressure : 0}
+            minValue={0}
+            maxValue={maxPressure}
+            alertAfter={maxPressure * 0.70}
+            ranges={{
+              "good": [0, maxPressure * 0.70],
+              "average": [maxPressure * 0.70, maxPressure * 0.85],
+              "bad": [maxPressure * 0.85, maxPressure],
+            }}
+            format={formatPressure}
+          />
+        </LabeledList.Item>
+      </LabeledList>
+    </Section>
+  );
+};
+
+export const TTV = (_props, context) => {
+  const { act, data } = useBackend<AirVendorParams>(context);
+  const {
+    opened,
+    tank_one,
+    tank_two,
+  } = data;
+  let windowWidth = 650;
+  let windowHeight = 160;
+  return (
+    <Window width={windowWidth} height={windowHeight}>
+      <Window.Content>
+        <Stack>
+          <Stack.Item width={windowWidth/3}>
+            <TankInfo height={windowHeight/16.666} tank={tank_one} />
```

Comment:
I don't think this `height` param is doing anything

Reply:
it does but i can define a constant height tbh ive had to manually edit it anyway since height var is weird

Reply:
removed height/width variables